x/evm/keeper/genesis.go

```diff
@@ -43,6 +43,10 @@ func (k baseKeeper) InitGenesis(ctx sdk.Context, state types.GenesisState) {
 		for _, token := range chain.Tokens {
 			ck.setTokenMetadata(ctx, token)
 		}
+
+		if chain.LatestBatchedCommandsID != nil {
```

Comment:
The genesis state is not valid if `len(CommandBatches) > 0` and `LatestBatchedCommandsID == nil`.

Reply:
removed this field and now the latest batch is calculated during import